docs/tutorials/classification.ipynb

```diff
@@ -0,0 +1,388 @@
+    ")\n",
+    "plt.gca().set_aspect(\"equal\")\n",
+    "plt.axhline(0, color=\"k\")\n",
+    "plt.axvline(0, color=\"k\")\n",
+    "plt.xlabel(r\"$x_{1}$\")\n",
+    "plt.ylabel(r\"$x_{2}$\")\n",
+    "plt.legend()\n",
+    "_ = plt.title(\"XOR Dataset\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xs = jnp.linspace(-2, 2, num=100)\n",
+    "ys = jnp.linspace(-2, 2, num=100)\n",
+    "\n",
+    "xx, yy = jnp.meshgrid(xs, ys)\n",
+    "xx = xx.T\n",
+    "yy = yy.T\n",
+    "true_X = jnp.vstack((xx.ravel(), yy.ravel())).T\n",
+    "true_y = jnp.logical_xor(true_X[:, 0] > 0, true_X[:, 1] > 0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def sigmoid(x):\n",
+    "    return 1 / (1 + jnp.exp(-x))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As the likelihood is non-Gaussian we need to use Markov Chain Monte Carlo (MCMC) or Variational Inference (VI) to marginalize numerically.  \n",
+    "This follows from the example in {ref}`likelihoods-mcmc` and {ref}`modeling-numpyro`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import jax\n",
+    "import jax.numpy as jnp\n",
+    "from flax.linen.initializers import zeros\n",
+    "import numpyro\n",
+    "import numpyro.distributions as dist\n",
+    "from tinygp import kernels, GaussianProcess\n",
+    "\n",
+    "jax.config.update(\"jax_enable_x64\", True)\n",
+    "\n",
+    "\n",
+    "def model(x, y=None):\n",
+    "    # The parameters of the GP regression\n",
+    "    mean = numpyro.param(\"mean\", jnp.zeros(()))\n",
+    "    sigma = numpyro.param(\"sigma\", jnp.ones(()))\n",
+    "    ell = numpyro.param(\"ell\", jnp.ones(()))\n",
+    "\n",
+    "    # Set up the kernel and GP objects\n",
+    "    kernel = (sigma**2) * kernels.ExpSquared(scale=ell)\n",
+    "    gp = GaussianProcess(kernel, x, diag=1e-5, mean=mean)\n",
+    "\n",
+    "    gp_out = numpyro.sample(\"gp_out\", gp.numpyro_dist())\n",
+    "    # Squashing the GP regression real output to the [0, 1] range\n",
+    "    # using sigmoid as the link function\n",
+    "    p = sigmoid(gp_out)\n",
+    "\n",
+    "    # Finally our observation model is Bernoulli distribution\n",
+    "    # where 'p' is the probability of Class 2\n",
+    "    numpyro.sample(\"obs\", dist.Bernoulli(probs=p), obs=y)\n",
+    "\n",
+    "    if y is not None:\n",
+    "        # Posterior Inference on true_X input values\n",
+    "        numpyro.deterministic(\"pred\", gp.condition(gp_out, true_X).gp.loc)\n",
+    "\n",
+    "\n",
+    "nuts_kernel = numpyro.infer.NUTS(model, target_accept_prob=0.8)\n",
+    "mcmc = numpyro.infer.MCMC(\n",
+    "    nuts_kernel,\n",
+    "    num_warmup=1000,\n",
+    "    num_samples=1000,\n",
```

Comment:
It's currently taking too long to build.
Should I reduce the num_samples, and num_warmup (to 500)?